src/index.html

```diff
@@ -8,7 +8,60 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
+<<<<<<< HEAD
```

Comment:
It seems you are having a merge conflicts.

There is some documentation to resolve the conflict: https://help.github.com/en/articles/resolving-a-merge-conflict-on-github

Do reach out for help when you meet us.